scripts/build.mjs

```diff
@@ -55,7 +55,7 @@ async function run(argv) {
     );
   }
 
-  const babelConfigPath = path.resolve(getWorkspaceRoot(), 'babel.config.js');
+  const babelConfigPath = path.resolve(getWorkspaceRoot(), 'babel.config.mjs');
```

Comment:
Will we still need `scripts/build.mjs`?

Reply:
I've kept it for now. Once all the repo PRs are merged, we can delete it in a separate PR.

Reply:
but these may not necessarily have babel.config.mjs yet